Data/Sys/GameSettings/RODE01.ini

```diff
@@ -25,6 +25,5 @@ PH_ExtraParam = 0
 PH_ZNear =
 PH_ZFar =
 
-[Video_Hacks]
-EFBEmulateFormatChanges = True
-
+[Video_Settings]
+AspectRatio = 2
```

Comment:
This game only output at 4:3 (regardless of whichever aspect ratio is set in the Wii tab). The default Dolphin setting in Wii tab is 16:9 and the default AR in graphics config is Auto, making this game output stretched at 16:9.
About the format changes, this was an error at my side, I'm aware of it and was not supposed to remove this entry, only enforcing the correct AR. I'll fix that